src/components/NavigationComponent.svelte

```diff
@@ -0,0 +1,58 @@
+<script>
+  import { buildHyperlink } from "../helpers/buildHyperlinkHelper";
+  import { selectedGeographyStore } from "../stores/stores";
+
+  export let topicSlug, search, currentURL, onClick;
+  export let isMobile = false;
+
+  $: currentLocationLink = search ? `${currentURL}${search}` : currentURL;
+</script>
+
+<div class={`ons-grid ons-grid--flex navigation-container ${isMobile ? "mobile" : ""}`}>
+  <div class="ons-grid__col ons-col-auto ons-grid__col--flex divider">
+    <div class="ons-pl-grid-col">
+      <a
+        href={buildHyperlink({
+          topic: topicSlug,
+          selectedGeography: $selectedGeographyStore,
+        })}>New category</a
+      >
+    </div>
+  </div>
+  <div class="ons-grid__col ons-col-auto ons-grid__col--flex divider">
+    <div class="ons-pl-grid-col">
+      <a href={currentLocationLink} on:click={onClick}> New location </a>
+    </div>
+  </div>
+  <div class="ons-grid__col ons-col-auto ons-grid__col--flex">
+    <div class="ons-pl-grid-col">
+      <a href={buildHyperlink({ selectedGeography: $selectedGeographyStore })}>Back to start</a>
+    </div>
+  </div>
+</div>
+
+<style lang="scss">
+  @import "../../node_modules/@ons/design-system/scss/vars/_index.scss";
+
+  .navigation-container {
+    min-height: 46px;
+    background-color: $color-ocean-blue;
+    margin: unset;
+    padding: 6px 0px;
+  }
+  .mobile {
```

Comment:
We should use breakpoints here rather than `isMobile` but I'll take care of it when moving from Sass to PostCSS so all fine as well. 😊